### PR TITLE
feat: Add webpack

### PR DESCRIPTION
**Pull request by tofumatt**
🎉

### CODE REVIEW

File: themes/kelpie/_site-header.scss [hunk 1]
```diff
@@ -1,8 +0,0 @@
-/*--------------------------------------------------------------
-# Site Footer
```
Comment (tofumatt) [inline themes/kelpie/_site-header.scss:2]: I don't think this file was in use. If it was, it should be moved! 😆

File: themes/kelpie/assets/scss/style.scss [hunk 2]
```diff
@@ -20,68 +20,21 @@ Normalizing styles have been helped along thanks to the fine work of
...
-@import "base/wp-normalize";
+// Start with nice, clean base styles.
+@import 'base/normalize';
+@import 'base/sanitize';
+@import 'base/wp-normalize';
 
 // Import all our variables & design tokens.
-@import "design-tokens/variables";
-@import "design-tokens/typography";
+@import 'design-tokens/**/*.scss';
```
Comment (tofumatt) [inline themes/kelpie/assets/scss/style.scss:31]: If you aren't cool with this we can revert it, but it seems WAY less annoying…

Comment (sarahmonster) [reply themes/kelpie/assets/scss/style.scss:31]: OH MY GOD YOU CAN DO THIS?!?!?!

File: themes/kelpie/_site-header.scss (hunk 1 above)
Comment (sarahmonster) [reply themes/kelpie/_site-header.scss:2]: Yes yes yes thank you. I keep accidentally opening it, wondering why it exists, and then losing track and not being able to find where it is. 🙄

File: themes/kelpie/assets/scss/style.scss (hunk 2 above)
Comment (tofumatt) [reply themes/kelpie/assets/scss/style.scss:31]: I had to add https://github.com/octopusthink/kelpie/pull/2/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR4 but yeah. 😄

**Comment by sarahmonster** [comment]
Uh, so apparently I just totally piggybacked on your PR here. It's working for me! Let's go ahead and merge, and then I can go back to committing to master like some kind of cowboy. 🤠

Thanks so much for this! Seems to work well and I LOVE not having to manually specify all the SCSS files to import. ❤️

**Pull request merged**